docs/source/managed-federation/deployment.mdx

```diff
@@ -212,38 +192,38 @@ extend type Product @key(fields: "upc") {
 
 ## Reliability and security
 
 The gateway's configuration is exposed through [Google Cloud Storage (GCS)](https://cloud.google.com/storage). For each Studio [graph API key](https://www.apollographql.com/docs/studio/api-keys/#graph-api-keys) with a [permission level](https://www.apollographql.com/docs/studio/org/members/#graph-key-roles) greater than `Consumer`, Apollo Studio provisions a public file that's accessible via the hash of that key. In the event that your configuration is inaccessible due to an outage in GCS, the gateway continues to serve the last fetched configuration.
 
 If the Apollo Studio API experiences an outage, changes to your configuration will halt until the outage resolves, but your most recently published configuration for each variant is still accessible via GCS.
 
-## The `service:push` lifecycle
+## The `subgraph publish` lifecycle
 
-Whenever you call `apollo service:push` for an implementing service, it both updates that service's registered schema and updates the gateway's managed configuration. Because the graph is dynamically changing and multiple services might be updated simultaneously, it's possible for changes to cause composition errors, even if `apollo service:check` was successful. For this reason, updating a federated service will re-trigger composition in the cloud, ensuring that the federated services still compose to form a complete graph before provisioning the configuration. The workflow behind the scenes can be summed up as follows:
+Whenever you call `rover subgraph publish` for a particular subgraph, it both updates that subgraph's registered schema and updates the gateway's managed configuration. Because the graph is dynamically changing and multiple subgraphs might be updated simultaneously, it's possible for changes to cause composition errors, even if `rover subgraph check` was successful. For this reason, updating a subgraph re-triggers composition in the cloud, ensuring that all subgraphs still compose to form a complete supergraph before provisioning the configuration. The workflow behind the scenes can be summed up as follows:
 
-1. The partial schema is uploaded to a secure location and indexed.
-2. The federated service is updated in the registry to point to the partial schema.
-3. All federated services are composed in the cloud to produce a new complete schema.
+1. The subgraph schema is uploaded to a secure location and indexed.
+2. The subgraph is updated in the registry to point to the new schema.
+3. All subgraphs are composed in the cloud to produce a new supergraph schema.
 4. If composition fails, the command exits and emits errors.
-5. If composition succeeds, the top-level configuration file is updated in-place to point to the updated set of federated services.
+5. If composition succeeds, the top-level configuration file is updated in-place to point to the updated set of subgraphs.
```

Comment:
Nuance: in the old model we'd publish to the cloud the full set of subgraphs, but in the new model we publish just the supergraph, itself. Might be worth clarifying that here?